sshtunnel.cpp

```diff
@@ -84,11 +84,20 @@ bool SshTunnel::connect(const Glib::ustring &server, const Glib::ustring &userna
     if (!verify_host())
         return false;
 
+    ssh_userauth_none(m_ssh, nullptr);
```

Comment:
Should probably check the return code here:
```suggestion
    if (ssh_userauth_none(m_ssh, nullptr) == SSH_AUTH_SUCCESS)
        return true;
```